crypto/keys/pkcs11/no_pkcs11.go

```diff
@@ -1,3 +1,4 @@
+//go:build !pkcs11
```

Comment:
Are we keeping both formats only for compatibility reasons, or the new tooling wants both lines?

Reply:
This comment didn't appear until I pressed merge. What fun. Luckily, the answer is that the new tooling wants both lines. The old one will be deprecated in some future version of go.

Reply:
SGTM.